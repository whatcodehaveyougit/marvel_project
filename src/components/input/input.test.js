import { render } from '@testing-library/react';
import CustomInput from './input.component';


describe('CustomInput', () => {
  it('renders correctly', () => {
    const tree = render(<CustomInput />).asFragment();
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with props', () => {
    const tree = render(<CustomInput type='text' name='test' placeholder='test' />).asFragment();
    expect(tree).toMatchSnapshot();
  });
})