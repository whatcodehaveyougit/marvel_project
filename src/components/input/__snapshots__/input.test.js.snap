// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <input
    value=""
  />
</DocumentFragment>
`;

exports[`renders correctly with props 1`] = `
<DocumentFragment>
  <input
    name="test"
    placeholder="test"
    type="text"
    value=""
  />
</DocumentFragment>
`;
